src/main/java/it/gov/pagopa/ranking/model/BeneficiaryRankingStatus.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.ranking.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum BeneficiaryRankingStatus {
+
+    ELIGIBLE_OK("ELIGIBLE_OK"),
+    ELIGIBLE_KO("ELIGIBLE_KO"),
```

Comment:
have these values been shared with the FE team?
probably we should also add the ONBOARDING_KO status

CC: @StefanoTerragni

Reply:
Yes, all these values has been reviewed today with them.

Reply:
@StefanoTerragni do you agree to add ONBOARDING_KO status?

Reply:
@antonioT90 Correct, we expect it to be taken into consideration from the admissibility component and use it as a new value to be forwarded to FE interface